staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -574,6 +574,8 @@ type VirtualMachineInstanceNetworkInterface struct {
 	InfoSource string `json:"infoSource,omitempty"`
 	// Specifies how many queues are allocated by MultiQueue
 	QueueCount int32 `json:"queueCount,omitempty"`
+	// Ready specifies if the corresponding pod interface is properly configured by CNI
+	Ready bool `json:"ready,omitempty"`
```

Comment:
I think this should be introduced when you actually use it.
At this stage you cannot even reason for it in the commit and it is after all affecting all interfaces/networks.

I have a small problem with the generic name itself, as it may be interpreted in many ways by readers and can be confusing. It may also hijack the name for other "ready" states in the future.
As an example, while we call it "ready" here, it may not yet be "ready" in the guest itself (i.e. plugged into the domain and recognized by the guest).

Here some options:
- `CNIReady`
- `CNISync`
- `PodConfigDone`

Reply:
I am OK with any of the naming alternatives. Going with `PodConfigDone` unless someone stops me, since to me it hits the abstraction sweet spot - CNI is just an implementation detail, when all things are considered.

I'll think on how I can move this around to other commit.

Reply:
Resolving since I've renamed the attribute, and moved its implementation to the "correct" commit.